hnn_core/gui/_viz_manager.py

```diff
@@ -99,6 +99,35 @@ def plot_type_coupled_change(new_plot_type, target_data_selection):
         target_data_selection.disabled = False
 
 
+def unlink_relink(attribute: str):
```

Comment:
we don't do type annotations ... I think it will have to be a repo-wide change, let's stick to repo conventions for now to keep things consistent